--- a/lib-core.sh
+++ b/lib-core.sh
@@ -290,20 +290,20 @@
 
     case "${global_param}" in
       -d)
-        if [[ "$(readlink -m "${value}")" =~ "${REPO_DIR}" ]]; then
-          prompt -e "'${display_param}' ERROR: Can't install in the source directory."
-          has_any_error="true"
-        elif [[ ! -w "${value}" && ! -w "$(dirname "${value}")" ]]; then
-          prompt -e "'${display_param}' ERROR: You have no permission to access that directory."
-          has_any_error="true"
-        else
+        #if [[ "$(readlink -m "${value}")" =~ "${REPO_DIR}" ]]; then
+        #  prompt -e "'${display_param}' ERROR: Can't install in the source directory."
+        #  has_any_error="true"
+        #elif [[ ! -w "${value}" && ! -w "$(dirname "${value}")" ]]; then
+        #  prompt -e "'${display_param}' ERROR: You have no permission to access that directory."
+        #  has_any_error="true"
+        #else
           if [[ ! -d "${value}" ]]; then
             prompt -w "Destination directory does not exist. Let's make a new one..."; echo
             mkdir -p "${value}"
           fi
 
           dest="${value}"
-        fi
+        #fi
 
         remind_relative_path "${display_param}" "${value}"; variant_found="skip" ;;
       -b)
--- a/install.sh
+++ b/install.sh
@@ -144,4 +144,5 @@
   echo; prompt -w "${final_msg}"; echo
 
   [[ -x /usr/bin/notify-send ]] && notify-send "'${name}' theme has been installed. Enjoy!" "${notif_msg}" -i "dialog-information-symbolic"
+  exit 0
 fi
