--- a/lib-core.sh
+++ b/lib-core.sh
@@ -43,7 +43,8 @@
   SED_OPT="-i"
 fi
 
-SUDO_BIN="$(which sudo)"
+#SUDO_BIN="$(which sudo)"
+SUDO_BIN=""
 
 #------------Directories--------------#
 THEME_SRC_DIR="${REPO_DIR}/src"
@@ -223,7 +224,7 @@
 fi
 
 rm -rf "${WHITESUR_TMP_DIR}"
-mkdir -p "${WHITESUR_TMP_DIR}"; exec 2> "${WHITESUR_TMP_DIR}/error_log.txt"
+mkdir -p "${WHITESUR_TMP_DIR}"; #exec 2> "${WHITESUR_TMP_DIR}/error_log.txt"
 
 signal_exit() {
   rm -rf "${WHITESUR_TMP_DIR}"
@@ -467,20 +468,20 @@
 
     case "${global_param}" in
       -d)
-        if [[ "$(readlink -m "${value}")" =~ "${REPO_DIR}" ]]; then
-          prompt -e "'${display_param}' ERROR: Can't install in the source directory."
-          has_any_error="true"
-        elif [[ ! -w "${value}" && ! -w "$(dirname "${value}")" ]]; then
-          prompt -e "'${display_param}' ERROR: You have no permission to access that directory."
-          has_any_error="true"
-        else
+        #if [[ "$(readlink -m "${value}")" =~ "${REPO_DIR}" ]]; then
+        #  prompt -e "'${display_param}' ERROR: Can't install in the source directory."
+        #  has_any_error="true"
+        #elif [[ ! -w "${value}" && ! -w "$(dirname "${value}")" ]]; then
+        #  prompt -e "'${display_param}' ERROR: You have no permission to access that directory."
+        #  has_any_error="true"
+        #else
           if [[ ! -d "${value}" ]]; then
             prompt -w "Destination directory does not exist. Let's make a new one..."; echo
             mkdir -p "${value}"
           fi
 
           dest="${value}"
-        fi
+        #fi
 
         remind_relative_path "${display_param}" "${value}"; variant_found="skip" ;;
       -b)
@@ -683,7 +684,7 @@
 full_sudo() {
   if [[ ! -w "/root" ]]; then
     prompt -e "ERROR: '${1}' needs a root priviledge. Please run this '${0}' as root"
-    has_any_error="true"
+    #has_any_error="true"
   fi
 }
 
--- a/install.sh
+++ b/install.sh
@@ -163,6 +163,7 @@
 
   echo; prompt -w "${final_msg}"
   [[ -x /usr/bin/notify-send ]] && notify-send "'${name}' theme has been installed. Enjoy!" "${notif_msg}" -i "dialog-information-symbolic"
+  exit 0
 fi
 
 echo
